ui/components/ui/metafox-logo/horizontal-logo.js

```diff
@@ -0,0 +1,961 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const LOGO_WIDTH = 162;
+const LOGO_HEIGHT = 30;
```

Comment:
should we pass this as `props`?

Reply:
I did so at first but this is the only place we use this image so we can just use the lone constant for now.